free-programming-books-pt_BR.md

```diff
@@ -218,6 +218,7 @@
 
 * [Node.js para Leigos](http://udgwebdev.com/nodejs/)
 * [Raspagem de dados com Node.js](http://tableless.com.br/raspagem-de-dados-com-node-js/)
+* [Aplicações Web real time] (https://github.com/caio-ribeiro-pereira/livro-nodejs)
```

Comment:
This should be slotted in alphabetical order instead.
Also please remove the space between the ] and ( symbols so that it becomes a valid markdown link.

It'd be better if you wrote the full title of the book as displayed there which is: "Aplicações web real-time com Node.js"

Also please add the author's name after the book ie: `- Caio Ribeiro Pereira`